Jenkinsfile

```diff
@@ -1,170 +1,163 @@
-import groovy.json.JsonSlurper
 // This Jenkinsfile is used by Jenkins to run the BioModels step of Reactome's release.
-// It requires that the OrthoinferenceStableIdentifierHistory step has been run successfully before it can be run.
-def currentRelease
-def previousRelease
+// It requires that the OrthoinferenceStableIdentifierHistory and AddLinks-Insertion steps have been run successfully before it can be run.
+
+import org.reactome.release.jenkins.utilities.Utilities
+
+// Shared library maintained at 'release-jenkins-utils' repository.
+def utils = new Utilities()
+
 pipeline {
 	agent any
 
 	stages {
-  		// This stage checks that an upstream project, AddLinks-Insertion, was run successfully for its last build.
-		stage('Check OrthoinferenceStableIdentifierHistory build succeeded'){
+  		// This stage checks that two upstream projects, OrthoinferenceStableIdentifierHistory and AddLinks-Insertion, were run successfully for their last build.
+		stage('Check OrthoinferenceStableIdentifierHistory  and AddLinks-Insertion builds succeeded'){
```

Comment:
Nit pick :) - double space just before "and"?

Reply:
:roll_eyes:

Reply:
Nice catch